helpers/cache/redis.ts

```diff
@@ -0,0 +1,31 @@
+"use server";
+import { Redis } from "@upstash/redis"
+import dotenv from "dotenv";
+dotenv.config();
+
+
+const redis = new Redis({
+  url:  process.env.REDIS_URL!,
+  token: process.env.REDIS_TOKEN!,
```

Comment:
Can you add those environment variable to the example file at the root of the project ?
Could you also add a docker compose environment, or a pod with a redis container with podman ?

Reply:
At some point maybe but honsetly upstash is really cheap so might aswell usw that.